src/labkey/Domain.ts

```diff
@@ -467,6 +556,44 @@ export function getProperties(config: GetPropertiesOptions): XMLHttpRequest {
     });
 }
 
+export interface PropertyUsagesParams {
+    maxUsageCount?: number
+    propertyIds?: [number]
+    propertyURIs?: [string]
+}
+
+export interface PropertyUsagesResponse {
```

Comment:
I would recommend declaring a separate type for the usages. Something like:

```ts
export interface PropertyUsage {
    propertyId: number
    propertyURI: string
    usageCount: number
    objects: LineageItemBase[]
}

export interface PropertyUsagesResponse {
    data: PropertyUsage[]
}
```

Additionally, you could consider responding with an array of usages directly using a response transformer. Something like:

```ts
export interface PropertyUsage {
    propertyId: number
    propertyURI: string
    usageCount: number
    objects: LineageItemBase[]
}

// Note: The response is a PropertyUsage[]
export interface PropertyUsagesOptions extends PropertyUsagesParams, RequestCallbackOptions<PropertyUsage[]> {
    containerPath?: string
}

export function getPropertyUsages(config: PropertyUsagesOptions): XMLHttpRequest {
    const params: PropertyUsagesParams = {
        maxUsageCount: config.maxUsageCount
    };

    if (config.propertyIds)
        params.propertyIds = config.propertyIds;

    if (config.propertyURIs)
        params.propertyURIs = config.propertyURIs;

    return request({
        url: buildURL('property', 'propertyUsages.api', config.containerPath),
        params,
        // See the "responseTransformer" applied here
        success: getCallbackWrapper(getOnSuccess(config), config.scope, false, (json: any) => json.data),
        failure: getCallbackWrapper(getOnFailure(config), config.scope, true)
    })
}
```